extension/src/setup/index.ts

```diff
@@ -310,13 +337,6 @@ export class Setup
     }
   }
 
-  private async initializeDvc() {
```

Comment:
[F] Registered externally and used in that way so we get a "free" telemetry event